osx/KBKit/Podfile.lock

```diff
@@ -82,4 +82,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 63b8f4d71cbba2bc6adfb7e369a31a74bbbf7f8b
 
-COCOAPODS: 1.2.0
+COCOAPODS: 1.4.0
```

Comment:
Is this because of the Xcode7 -> Xcode8 change?

Reply:
The build instructions have you do:

```
sudo gem install cocoapods
pod setup
```

So I think it's probably related to the version of cocoapods.  Ideally we wouldn't have any generated code checked in, but I'm not sure how to tackle that yet.